dsl/src/test/scala/org/http4s/dsl/ResponseGeneratorSpec.scala

```diff
@@ -1,34 +1,42 @@
 package org.http4s
 package dsl
 
-import cats.data._
+import cats._
+import cats.effect.IO
 import org.http4s.headers.{Accept, `Content-Length`, `Content-Type`}
 
 class ResponseGeneratorSpec extends Http4sSpec {
 
   "Add the EntityEncoder headers along with a content-length header" in {
-    val body = "foo"
-    val resultheaders = Ok(body)(EntityEncoder.stringEncoder).unsafeRun.headers
-    EntityEncoder.stringEncoder.headers.foldLeft(ok) { (old, h) =>
+    val body          = "foo"
+    val resultheaders = Ok(body)(Applicative[IO], FlatMap[IO], EntityEncoder.stringEncoder[IO]).unsafeRunSync.headers
```

Comment:
No, not in this case. However, if they aren't there it has to be `Ok[IO, String](body)`.

In the other test cases in that file an explicit encoder is used so the type class instances have to be supplied manually.

Would be nice with multiple implicit parameter lists here.